### ISSUE TITLE
Chat server with avatars example code in documentation fails.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4459]
|<img alt="mdamen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| mdamen reported|
|-|-|
|Trac ID|trac#4459|
|Type|defect|
|Created|2010-05-26 19:19:23Z|
|Branch|https://github.com/twisted/twisted/tree/pb-chat-example-4459|

You can run the doc/core/howto/listings/pb/chatserver.py and doc/core/howto/listings/pb/chatclient.py to reproduce the problem.

<details><summary>Searchable metadata</summary>

```
trac-id__4459 4459
type__defect defect
reporter__mdamen mdamen
priority__lowest lowest
milestone__ 
branch__branches_pb_chat_example_4459 branches/pb-chat-example-4459
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__pb pb
keywords__ 
time__1274901563000000 1274901563000000
changetime__1275686918000000 1275686918000000
version__None None
owner__ 
cc__ivank
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Please be specific.  Updating description to include full paths of the examples in question.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [29134]) Branching to 'pb-chat-example-4459'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|ivank set owner to @exarkun|
|-|-|

I manually tested the change and it did fix a bug. The changes look good, please merge. I think I found another bug in the process of testing: [#4488](https://github.com/twisted/twisted/issues/4488)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [29186]) Merge pb-chat-example-4459

Author: exarkun
Reviewer: ivank
Fixes: [#4459](https://github.com/twisted/twisted/issues/4459)

Fix one of the PB examples so that it can actually call the remote methods it
wants to call, rather than failing with an unhandled exception.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
